src/scripts/main.js

```diff
@@ -1,7 +1,438 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const buttonStart = document.querySelector('.button');
+const messageStart = document.querySelector('.message-start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const cells = document.querySelectorAll('.field-cell');
+const score = document.querySelector('.game-score');
+const addScore = document.querySelector('.add-points');
 
-// Write your code here
+const twoOrFour = () => {
+  let value;
+
+  if (Math.random() <= 0.1) {
+    value = 4;
+  } else {
+    value = 2;
+  }
+
+  return value;
+};
+
+const randomNewCell = () => {
+  let rando;
+
+  do {
+    rando = Math.floor(Math.random() * cells.length);
+  } while (cells[rando].textContent !== '');
+
+  return rando;
+};
+
+const cellStyle = () => {
+  cells.forEach(cell => {
+    cell.classList.remove(
+      'field-cell--2', 'field-cell--4', 'field-cell--8',
+      'field-cell--16', 'field-cell--32', 'field-cell--64',
+      'field-cell--128', 'field-cell--256', 'field-cell--512',
+      'field-cell--1024', 'field-cell--2048',
+    );
+
+    switch (cell.textContent) {
+      case '':
+        break;
+      case '2':
+        cell.classList.add('field-cell--2');
+        break;
+      case '4':
+        cell.classList.add('field-cell--4');
+        break;
```

Comment:
```suggestion
      case '2':
      case '4':
        cell.classList.add(`field-cell--${cell.textContend}`);
        break;
```
you may join all cases (except 2048) like this